templates/node--localgov-news-article--full.html.twig

```diff
@@ -67,23 +66,41 @@
  */
 #}
 
-<article{{ content_attributes.addClass('content').addClass('newsroom--article') }}>
-  <div>
-    <div class="newsroom--info">
-      <span class="newsroom--date">{{ node.localgov_news_date()|date("j F Y") }}</span>
+{%
+  set classes = [
+  'news-article',
+  'node',
+  'node--type-' ~ node.bundle|clean_class,
+  node.isPromoted() ? 'node--promoted',
+  node.isSticky() ? 'node--sticky',
+  not node.isPublished() ? 'node--unpublished',
+  view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
+]
+%}
 
-      {% if not node.localgov_news_categories.isEmpty() %}
-        <span class="newsroom--category">
-        <a href="{{ path('entity.taxonomy_term.canonical', {'taxonomy_term': node.localgov_news_categories.entity.id()}) }}">
-          {{ node.localgov_news_categories.entity.label() }}
-        </a>
-      </span>
-      {% endif %}
+<article{{ attributes.addClass(classes) }}>
 
-    </div>
+  <div class="news-article__metadata">
+    <time datetime="{{ node.localgov_news_date()|date("U") }}" class="news-article__metadata-item news-article__metadata--date">
+      {{ node.localgov_news_date()|date("j F Y") }}
```

Comment:
For this particular item, I'd like if we rendered the `localgov_news_date` field via the "Manage display" page with a default formatter of "j F Y". That would mean each site could override this without needed to maintain their own copy of this template. If we hardcode the date format in the template, then every theme must use the same format or else clone this template and maintain their own version of it.